src/functionalTest/resources/features/F-010 DELETE Linked Hearing Group/F-010.feature

```diff
@@ -0,0 +1,37 @@
+@F-010
+Feature: F-010: Delete linked hearing group
+
+  Background:
+    Given an appropriate test context as detailed in the test data source
+    Given a user with [an active profile in CCD]
+
```

Comment:
No error scenarios are here 
1. Check that the provided id is an extant/active linkedGroupDetails.requestId - return 404 if not
2. Confirm that the linkedGroupDetails.state is 'ACTIVE' - return 400 "007 group is in a <state> state" otherwise
3. For each current hearing request in the group, confirm that its state is valid for unlinking as per hearingLinkedGroup Principles #4, returning 400 "008 Invalid state for unlinking hearing request <hearingId>" if not.
4. Map the request to the LA request format - error if mandatory fields are missing
5. If the result is 4XX, log the error, delete the group record and return 400 "005 rejected by List Assist".
6. If the result is 500 (timeout), update the state of the group to 'ERROR' and return 400 "006 List Assist failed to respond"